### ISSUE TITLE
Unsupported selectors

### ISSUE THREAD

**Issue by bramus** [GoogleChromeLabs/container-query-polyfill#48]
I’m trying out the polyfill [in this demo](https://codepen.io/bramus/pen/BaxadOa/51e50eddc1036ca8fc82045d8491b43f) and ran into problems with this snippet in Firefox:

```css
@container (width >= 24rem) {
	.videocard > .container:has(.infobox, .player) {
		flex: 1;
	}
}
```

In short, the `:has()` selector was breaking things here because Firefox does not support it (yet). I had expected the polyfill to recover from this. 

```
Uncaught DOMException: Element.matches: '.videocard > .container:has()' is not a valid selector
```

What happens now in Firefox, is that the whole layout goes FUBAR: the polyfill loads, but the largest defined layout is used due to the code breaking. Once you size the `.resizer` to its smallest size (`20rem`), the layout fixes itself and takes one the smallest defined layout. However, upon making the `.resizer` bigger again, the layout remains the same.

In order to fix my demo, I used this snippet instead, which only uses selectors Firefox supports:

```css
@container (width >= 24rem) {
	.videocard > .container {
		flex: 1 !important;
	}
}
```

**Comment by devknoll** [comment]
Thanks for reporting this. I'll push a fix in the morning to prevent unsupported selectors from breaking the update process.

**Comment by devknoll** [comment]
Fixed by [8eb2d82](https://github.com/GoogleChromeLabs/container-query-polyfill/commit/8eb2d828f72a7d3c906e43e58f3ba0f6aacf0d14)
